packages/apps/crm/src/screens/catalog/CatalogListScreen.js

```diff
@@ -86,61 +91,77 @@ const CatalogListScreen = ({navigation}) => {
 
   return (
     <Screen removeSpaceOnTop={true}>
-      <HeaderContainer
-        expandableFilter={false}
-        fixedItems={
-          <View style={styles.headerContainer}>
-            <AutoCompleteSearch
-              objectList={catalogList}
-              value={catalog}
-              onChangeValue={setCatalog}
-              fetchData={fetchCatalogFilter}
-              placeholder={I18n.t('Crm_Catalogs')}
-              oneFilter={true}
-              selectLastItem={false}
-            />
-          </View>
-        }
-        chipComponent={
-          <ChipSelect
-            mode="multi"
-            onChangeValue={chiplist => setSelectedStatus(chiplist)}
-            marginHorizontal={5}
-            selectionItems={
-              catalogTypeList
-                ? catalogTypeList.map((status, index) => {
-                    return {
-                      title: status.name,
-                      color: Catalog.getStatusColor(index, Colors),
-                      key: status.id,
-                    };
-                  })
-                : []
+      {crmConfig?.isManageCatalogs ? (
```

Comment:
Replace this by an if condition : 

if ( !crmConfig?.isManageCatalogs) {
 return warning screen
}

return normal screen